pkg/security/podsecuritypolicy/selinux/mustrunas_test.go

```diff
@@ -70,22 +72,27 @@ func TestMustRunAsGenerate(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error generating selinux %v", err)
 	}
-	if !reflect.DeepEqual(generated, opts.SELinuxOptions) {
+	if !reflect.DeepEqual(generated, &api.SELinuxOptions{
```

Comment:
reconstructing like this means we're vulnerable to drift if new fields are added. prefer converting to make sure we catch everything